nx-X11/programs/Xserver/hw/nxagent/NXevents.c

```diff
@@ -520,6 +520,37 @@ DefineInitialRootWindow(register WindowPtr win)
     }
 }
 
+#ifdef NXAGENT_SERVER
+extern Bool nxagentWMIsRunning;
+
+int
+EventSelectForWindow(register WindowPtr pWin, register ClientPtr client, Mask mask)
```

Comment:
please rename to nxagentEventSelectForWindow and let it be called from whereever needed.